app/assets/stylesheets/wcag.scss

```diff
@@ -280,7 +286,46 @@ html {
     color: var(--primary-medium);
   }
 
-  // Admin
+  .alert.alert-success a {
+    color: dark-light-choose($tertiary, $quaternary-high);
+  }
+
+  // User Preference
+  .user-menu.revamped .quick-access-panel li.unread,
+  .user-menu.revamped .quick-access-panel li.pending {
+    background-color: var(--tertiary-very-low);
+  }
+
+  .user-main .deleted-posts {
+    color: var(--primary);
+  }
+
+  .uploaded-image-preview
+    .image-upload-controls
+    .image-uploader-lightbox-btn
+    .d-icon {
+    color: var(--primary);
+  }
+
+  .settings .setting .setting-value .uploaded-image-preview {
+    background-color: var(--primary-very-low);
+  }
+
+  // Notifications
+
+  .user-menu.revamped .tabs-list .btn .badge-notification {
+    background-color: dark-light-choose($tertiary-high, $tertiary);
```

Comment:
I believe using `dark-light-choose` here can be problematic, because the SCSS color variables in this stylesheet may not be coming from the correct color scheme. One way to test this is to enable auto dark mode locally, switch the operating system to dark colors and observe the colors. 

Our usual workaround for this has been to add these sorts of choices to `color_transformations.scss` and `color_definitions.scss`.

Reply:
Ah, my first comment here is incorrect. The wcag stylesheet is only loaded with color schemes, so in fact, `dark-light-choose` will work as expected.